downstream/modules/platform/con-gs-developer-tools.adoc

```diff
@@ -2,16 +2,5 @@
 
 = {ToolsName}
 
-{ToolsName} are an integrated and supported suite of capabilities that help IT practitioners at any skill level accurately create automation content faster than with manual coding. 
-This includes building, testing, and deploying modules and playbooks without requiring hard coding skills.
-
-These tools include:
-
-* Ansible Molecule - creates test environments that mimic real ones, allowing you to test playbooks, roles, and collections across many instances, operating systems, virtualization providers, and more.
-* Ansible Lint - analyzes your playbooks, roles, and collections—looking at the markup syntax and how you are using the modules. 
-It goes beyond regular YAML linters by checking Ansible tasks themselves. 
-* Execution environment builder - removes the risk of dependency issues when creating automation content by installing your collections and all of their dependencies in a containerized image (known as an {ExecEnvShort}).
-* Content testing suite - run checks to enforce coding standards and requirements, unit tests against Ansible modules, and integration tests to ensure workflows connect to your source code repositories.
-* Automation content navigator - reviews and explores collections, configurations, documentation, images, inventories, and playbooks. 
-Complete tasks such as watching jobs, sharing playbooks, browsing {ExecEnvShort} and inventories, and extracting reusable examples.
-* Ansible content development kit - is a bundled collection of tools that expands Ansible content creation from IT practitioners performing manual tasks to self-enabled automation content developers.
+{ToolsName} are an integrated and supported suite of capabilities that help IT practitioners at any skill level generate automation content faster than they might with manual coding. 
+{ToolsName} can help you create, test, and deploy automation content like playbooks, {ExecEnvShort}s, and collections quickly and accurately using recommended practices. For more information on how {ToolsName} can help you create automation content, see our documentation on link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.4/html/developing_ansible_automation_content/index[Developing Ansible automation content].
```

Comment:
You'll want to update this link to point to the new 2.5 version of the doc which has a different title. I suggest you use the attribute that Aine created. So it will be:

link:{LinkDevelopAutomationContent}.

Reply:
aha, I missed that one. Thanks!